includes/actions.php

```diff
@@ -371,3 +371,55 @@ function wpum_register_profile_privacy_fields() {
 	         ) );
 }
 add_action( 'carbon_fields_register_fields', 'wpum_register_profile_privacy_fields' );
+
+add_action( 'wp_ajax_wpum_upload_profile_image', 'wpum_upload_profile_image' );
```

Comment:
Getting a 400 error when uploading files on a registration form (user not logged in)

Reply:
@polevaultweb 
I'll check.

Reply:
@polevaultweb 
I've committed new changes for this.